node/full_node_integration_test.go

```diff
@@ -133,6 +133,64 @@ func TestTxGossipingAndAggregation(t *testing.T) {
 	}
 }
 
+func TestLazyAggregator(t *testing.T) {
```

Comment:
[non-blocking][question] What can we test in a follow-up issue to improve the testing surface area?